bidiClient/test_bidi.py

```diff
@@ -832,74 +858,127 @@ async def test_serialisation_date(websocket):
         "new Date('2021-02-18T13:53:00+0200')",
         {
             "type":"date",
-            "value":"Thu Feb 18 2021 11:53:00 GMT+0000 (Coordinated Universal Time)"},
-        True, websocket)
+            "value":"Thu Feb 18 2021 11:53:00 GMT+0000 (Coordinated Universal Time)",
+            "objectId":"__any_value__"},
+        websocket)
 
 @pytest.mark.asyncio
 async def test_serialisation_windowProxy(websocket):
     await assertSerialisation(
         "this.window",
-        {"type":"window"},
-        True, websocket)
+        {
+            "type":"window",
+            "objectId":"__any_value__"
+        },
+        websocket)
 
 @pytest.mark.asyncio
 async def test_serialisation_error(websocket):
     await assertSerialisation(
         "new Error('Woops!')",
-        {"type":"error"},
-        True, websocket)
+        {
+            "type":"error",
+            "objectId":"__any_value__"
+        },
+        websocket)
+
+# TODO: implement after serialisation MaxDepth logic specified:
+# https://github.com/w3c/webdriver-bidi/issues/86
+@pytest.mark.asyncio
+async def test_serialisation_array(websocket):
+    await assertSerialisation(
+        "[1, 'a', {foo: 'bar'}, [2,[3,4]]]",
+        {
+            "type":"array",
+            "objectId":"__any_value__",
+            "value":[{
+                "type":"number",
+                "value":1
+            },{
+                "type":"string",
+                "value":"a"
+            },{
+                "type":"object",
+                "objectId":"__any_value__"
+            },{
+                "type":"array",
+                "objectId":"__any_value__"}]},
+        websocket)
+
+# TODO: implement after serialisation MaxDepth logic specified:
+# https://github.com/w3c/webdriver-bidi/issues/86
+@pytest.mark.asyncio
+async def test_serialisation_object(websocket):
+    await assertSerialisation(
+        "{'foo': {'bar': 'baz'}, 'qux': 'quux'}",
+        {
+            "type":"object",
+            "objectId":"__any_value__",
+            "value":[[
+                "foo", {
+                    "type":"object",
+                    "objectId":"__any_value__"}],[
+                "qux", {
+                    "type":"string",
+                    "value":"quux"}]]},
+        websocket)
 
 # TODO: add `NodeProperties` after serialisation MaxDepth logic specified:
 # https://github.com/w3c/webdriver-bidi/issues/86.
 @pytest.mark.asyncio
 async def test_serialisation_node(websocket):
-    await assertSerialisation(
-        "document.createElement('p')",
-        {"type":"node"},
-        True, websocket)
+    contextID = await get_open_context_id(websocket)
+    await goto_url(websocket, contextID,
+        "data:text/html,<div style='background: red'><h2>test</h2></div>")
```

Comment:
```suggestion
        "data:text/html,<div id='test'><h2>test</h2></div>")
```

This won't be visible when running the tests, but still I'd suggest avoiding anything with the color red in tests, reserving that for indicating failure.

Reply:
replaced with even more abstract `some_attr_name='some_attr_value'`